pages/common/hub.md

```diff
@@ -11,6 +11,14 @@
 
 `hub clone {{github_username}}/{{repo_name}}`
 
+- Fork repository cloned from another user (creates a remote named after your GitHub username):
+
+`hub fork`
+
+- Create a pull request from your fork:
+
+`git push {{github_username}} && hub pull-request`
```

Comment:
I'm not sure combining these commands is wise. The first one is usual git workflow, but it threw me back for a bit because I wondered if it had a special meaning here. In fact, does github_username here mean that the previous command created a branch named after your GH username? Otherwise I'm not sure I follow what's going on here.